### PR TITLE
fix(sidebarConfig): resize after closing sidebar config

### PR DESCRIPTION
**Pull request by Tetrax-10**
Resize after closing sidebar config to its user specified size

Before:

https://user-images.githubusercontent.com/75513645/213998856-74910f72-075e-4f1c-b874-10e0f761402b.mp4

After:

https://user-images.githubusercontent.com/75513645/213998889-928904a5-fb4c-49c6-a8a2-54e970bf77cf.mp4

### CODE REVIEW

**Comment by Tetrax-10** [comment]
Sry for that formatting 😔 mb.
this is the correct formatting (use tabs). should i edit it?
```js
document.documentElement.style.setProperty(
    "--nav-bar-width",
    localStorage.getItem(`${(await Spicetify.CosmosAsync.get("sp://desktop/v1/session")).username}:nav-bar-width`) + "px"
);
```

File: jsHelper/sidebarConfig.js [hunk 1]
```diff
@@ -191,10 +191,14 @@
...
 	}
 
-	function removeInteraction() {
+	async function removeInteraction() {
 		hiddenList.classList.add("hidden-visually");
 		container.remove();
 		ordered.forEach(a => (a[0].onmouseover = undefined));
+		document.documentElement.style.setProperty(
+			"--nav-bar-width",
+			localStorage.getItem(`${(await Spicetify.CosmosAsync.get("sp://desktop/v1/session")).username}:nav-bar-width`) + "px"
```
Comment (kyrie25) [inline jsHelper/sidebarConfig.js:200]:
```suggestion
			localStorage.getItem(`${Spicetify.Platform.LocalStorageAPI.namespace}:nav-bar-width`) + "px"
```

File: jsHelper/sidebarConfig.js [hunk 2]
```diff
@@ -100,7 +100,7 @@
 
 	function writeStorage() {
 		const array = ordered.map(a => [a[0].dataset.id, a[1]]);
-		console.log(array);
+		// console.log(array); prints path
```
Comment (kyrie25) [inline jsHelper/sidebarConfig.js:103]: You can remove this statement altogether.

**Review by kyrie25 — CHANGES_REQUESTED** [review]
Just need to refactor some stuff.

File: jsHelper/sidebarConfig.js (hunk 2 above)
Comment (Tetrax-10) [reply jsHelper/sidebarConfig.js]: Lol never knew that existed 😂

**Comment by Tetrax-10** [comment]
Looks good

File: jsHelper/sidebarConfig.js [hunk 3]
```diff
@@ -191,10 +190,14 @@
 		}
 	}
 
-	function removeInteraction() {
+	async function removeInteraction() {
```
Comment (kyrie25) [inline jsHelper/sidebarConfig.js:193]: Revert this also since we're using static properties

**Review by kyrie25 — CHANGES_REQUESTED** [review]

File: jsHelper/sidebarConfig.js (hunk 3 above)
Comment (Tetrax-10) [reply jsHelper/sidebarConfig.js]: yeah just now did that totally forgot that one

**Review by kyrie25 — APPROVED** [review]
LGTM

**Review by theRealPadster — APPROVED** [review]

**Pull request merged**